sbin/build.sh

```diff
@@ -1080,6 +1080,17 @@ getFirstTagFromOpenJDKGitRepo() {
 
   if [ -z "$firstMatchingNameFromRepo" ]; then
     echo "WARNING: Failed to identify latest tag in the repository" 1>&2
+    if [ "${BUILD_CONFIG[DISABLE_ADOPT_BRANCH_SAFETY]}" == "true" ]; then
```

Comment:
Might need a comment as to why this logic is being added

Reply:
TL;DR Some version of https://github.com/adoptium/temurin-build/issues/2671 needs implemented.

Reply:
I've referenced that in the comments now